test/test.taskcontroller.js

```diff
@@ -23,7 +23,7 @@ describe('TaskController', function() {
     it('should throw an error if the priority is not valid.', function() {
       try {
         new TaskController('unknown');
-        assert.okay(false);
+        assert.ok(false);
```

Comment:
It turns out `assert.okay` is `undefined`:
<img width="449" alt="Screenshot 2025-04-09 at 12 50 29" src="https://github.com/user-attachments/assets/e29e1356-da0f-4941-988d-b6cdb6824c65" />

It could give us false negatives in tests. If you comment one line above, the test will still pass because this will throw `Uncaught TypeError: assert.okay is not a function` and we check if `e.name === 'TypeError'`